src/io/flutter/android/AndroidSdk.java

```diff
@@ -78,6 +95,16 @@ public static AndroidSdk fromEnvironment() {
   }
 
   /**
+   * Returns the best value of ANDROID_HOME to use.
+   * <p>
+   * If the given project has an Android SDK set, prefer that. Otherwise get it from the environment.
+   */
+  public static String chooseAndroidHome(@Nullable Project project) {
+    final AndroidSdk sdk = project == null ? null : fromProject(project);
+    return sdk == null ? EnvironmentUtil.getValue("ANDROID_HOME") : sdk.getHome().getPath();
```

Comment:
Oh hmmm, since values can change I guess this'd have to be method call.  Hmpfh.  Feel free to ignore.